packages/sync-actions/src/order-actions.js

```diff
@@ -94,6 +94,23 @@ function _buildDeliveryParcelsAction(
   return [addParcelActions, removeParcelActions]
 }
 
+function _buildDeliveryItemsAction(diffedItems, newDelivery = {}) {
+  const setDeliveryItemsAction = []
+  // If there is a diff it means that there were changes (update, adds or removes)
+  // over the items, which means that `setDeliveryItems` change has happened over
+  // the delivery
+  if (diffedItems && Object.keys(diffedItems).length > 0) {
+    setDeliveryItemsAction.push({
+      action: 'setDeliveryItems',
+      deliveryId: newDelivery.id,
+      deliveryKey: newDelivery.key,
```

Comment:
That's true, Originally i wanted to test this with one using `id` and the other the `key` but went finally just with ID, anyways I think is accurate anyways? If one of those is not present that property would be ignored, right?

Reply:
just assuming not pretty sure, that without id delivery doesnt have `id`, then it will remove and then add whole delivery
key is totally ignored https://github.com/commercetools/nodejs/blob/a9878e9b424bd7ac690e1804dcc26a2d50db6680/packages/sync-actions/src/utils/diffpatcher.js#L6-L11